polyply/src/build_file_parser.py

```diff
@@ -260,7 +260,7 @@ def _base_parser_geometry(tokens, _type):
         geometry_def["parameters"] = parameters
         return geometry_def
 
-def read_build_file(lines, molecules, topology):
+def read_build_file(lines, topology, molecules=None):
```

Comment:
I think we could even git rid of molecules there. Even though probably a lot of test fail then

Reply:
Couldn't there in the future be a case where you want to only apply certain bld file setting to specific molecules in a topology? (i.e. topologies of DNA + something else)